src/db/migrations/20220528044344_processedArtworks.ts

```diff
@@ -0,0 +1,19 @@
+import { Knex } from 'knex';
```

Comment:
i see u created this migration properly with timestamp - need to rename the other migrations to have timestamp too, as they're currently just in numeric order so could clash with timestamp ones - will do before merging this PR

Reply:
haha ye i saw, was going to ask if i need to make mine the custom numbers